src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -127,6 +127,10 @@ export class ProsemirrorAdapter {
         ];
     }
 
+    public disconnectedCallback() {
+        this.view.destroy();
```

Comment:
Whoa! Yeah, good catch!

I guess this means we have a memory leak in all apps that have the activities feed enabled with the text-editor… 😬

Reply:
This was merged in #3009